### ISSUE TITLE
Bugs in _custom_layers_and_blocks.py

### ISSUE THREAD

**Issue by tabmoo** [JanMarcelKezmann/TensorFlow-Advanced-Segmentation-Models#17]
Hello again!

`self.dropout = tf.keras.layers.Layer(dropout)` should be `self.dropout = tf.keras.layers.Dropout(dropout)` in lines 993 and 1027.

**Comment by JanMarcelKezmann** [comment]
Hi @tabmoo ,

sorry for my late reply.
Thank you for finding and telling me the bug, I have pushed a fix.
